README.md

```
@@ -1,6 +1,6 @@
-# GKE Info Go
+# <img align="left" width="45" height="45" src="https://github.com/user-attachments/assets/ce1fe535-f4f3-451f-bbd9-45fde04c000c">Istio Test
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)